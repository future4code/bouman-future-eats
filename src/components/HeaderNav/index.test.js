import React from 'react';
import renderer from 'react-test-renderer';
import { HeaderNav } from './index'

test('initial page snapshot', () => {
    const tree = renderer
        .create (<HeaderNav display="flex" menuText="Busca"/>)
        .toJSON();
    expect(tree).toMatchSnapshot()
  })