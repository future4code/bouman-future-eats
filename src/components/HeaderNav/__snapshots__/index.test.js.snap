// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`initial page snapshot 1`] = `
<div
  className="sc-bdVaJa gCACoJ"
>
  <img
    src="back.png"
    style={
      Object {
        "display": "flex",
      }
    }
  />
  <p
    className="sc-bwzfXH ghRyg"
  >
    Busca
     
  </p>
</div>
`;
